platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/network/TopologyTest.java

```diff
@@ -126,7 +126,8 @@ void testFullyConnectedTopology(final int numNodes, final long ignoredSeed) {
         final AddressBook addressBook =
                 RandomAddressBookBuilder.create(randotron).withSize(numNodes).build();
         for (int thisNode = 0; thisNode < numNodes; thisNode++) {
-            final NodeId outOfBoundsId = addressBook.getNextNodeId();
+            final NodeId outOfBoundsId =
+                    addressBook.getNodeId(addressBook.getSize() - 1).getOffset(1);
```

Comment:
Is there a reason not to use the `getNextAvailableNodeId` here as well? That would be more consistent with the rest of the tests, and would simplify any future refactoring involving this method.

Reply:
agreed